axum/src/routing/tests/mod.rs

```diff
@@ -641,6 +641,54 @@ async fn limited_body_with_streaming_body() {
     assert_eq!(res.status(), StatusCode::PAYLOAD_TOO_LARGE);
 }
 
+#[tokio::test]
+async fn extracting_state() {
```

Comment:
May be very nitpicky, but this reads much better to me:

```suggestion
async fn extract_state() {
```